client/stylesheets/_toCat.scss

```diff
@@ -5184,16 +5184,66 @@ On click the images will be displayed at normal size to complete the effect
 
 .headerSearchRadiusLogo {
     margin-right: 30px;
-    font-weight: 700;
+    font-weight: 400;
 }
 
 .headerSearchRadiusLogo a {
 	font-size: 1.3rem !important;
 }
 
+.headerSearchNavLogo {
+	margin-left: 16px;
+	margin-top: 3px;
+}
+
 /* End */
 
+/* Nav Search */
 
+.headerSearchNavField {
+	color: #404040;
+    font-weight: 400;
+    font-size: 0.9rem !important;
+}
+
+.headerSearchNavField i {
+	color: #404040 !important;
+	border-bottom: 2px solid #404040;
+    padding: 0 5px 6px 0;
+    margin-right: -5px;
+}
+
+.headerSearchNav {
+	outline: none !important;
```

Comment:
!important should not be used<br>Line should be indented with spaces, not tabs<br>Properties should be ordered -moz-appearance, -webkit-appearance, appearance, background-color, border, border-bottom, font-family, letter-spacing, margin-top, outline, padding-bottom